.tool-versions

```diff
@@ -1,3 +1,4 @@
 ruby 3.2.7
 postgres 15.8
 nodejs 23.6.0
+golang 1.24.0
```

Comment:
This isn't currently the go version used by the cli (`cli/go.mod` shows 1.22.2).  No objections to updating the go version, but we should make sure they match.

Reply:
I wonder why nothing complains. And weirdly best I can tell, `go mod` doesn't have a quick option to insert the current running go version. I wonder if it's an implied >=. Anyway, updated, for now